Assets/Scripts/UI/Pickup Pup/ScoutingDisplay/ScoutingDisplay.cs

```diff
@@ -0,0 +1,124 @@
+﻿/*
+ * Author(s): Isaiah Mann
+ * Description: Controls the logic of displaying dogs scounting
+ * Usage: [no notes]
+ */
+
+using UnityEngine;
+using System.Collections.Generic;
+
+public class ScoutingDisplay : PPUIElement 
+{
+	[SerializeField]
+	DogBrowser dogBrowser;
+	[SerializeField]
+	ScoutingReportUI scoutingReportDisplay;
+
+	DogOutsideSlot[] scoutingSlots;
+	Dictionary<int, DogOutsideSlot> slotsByIndex = new Dictionary<int, DogOutsideSlot>();
+
+	public void SendToSlot(Dog dog, int slotIndex)
+	{
+		DogOutsideSlot slot;
+		if(slotsByIndex.TryGetValue(slotIndex, out slot))
+		{
+			slot.ResumeScouting(dog);
+		}
+	}
+
+	public void SubscribeToTimerEnd(Dog dog)
+	{
+		dog.SubscribeToScoutingTimerEnd(handleScoutingTimerEnd);
```

Comment:
Yeah, the parent method is more for subscriptions on init. This one is meant to be used more dynamically